generator/dryrun.py

```diff
@@ -53,14 +75,50 @@ class Errors(Enum):
     PERMISSION_DENIED = 4
 
 
+class DryRunContext:
+    """DryRun builder class."""
+
+    def __init__(
+        self,
+        use_cloud_function=False,
+        id_token=None,
+        credentials=None,
+        dry_run_url=DRY_RUN_URL,
+    ):
+        """Initialize dry run instance."""
+        self.use_cloud_function = use_cloud_function
+        self.dry_run_url = dry_run_url
+        self.id_token = id_token
+        self.credentials = credentials
+
+    def init(
+        self,
+        sql=None,
+        project="moz-fx-data-shared-prod",
+        dataset=None,
+        table=None,
+    ):
+        """Initialize a DryRun instance."""
```

Comment:
I think naming this `init()` could be a bit confusing, as to me "init" implies modifying the object itself (the context in this case).  I'd suggest a name that sounds like it returns something (e.g. `create()`).